app/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/views/templates/SoftwareVendorsTemplateAlpha.scala.html

```diff
@@ -93,115 +91,43 @@
 
 }
 
-@softwareVendorCompatibility(vendor: SoftwareVendorModel) = @{
+@softwareSubmissionType(vendor: SoftwareVendorModel) = @{
 
   customSummaryListRow(
-    key = "software-compatibility",
-    filtersAvailable = vendor.orderedFilterSubset(VendorFilterGroups.compatibility)
-  )
-
-}
-
-@softwareVendorLanguage(vendor: SoftwareVendorModel) = @{
-
-  customSummaryListRow(
-    key = "language",
-    filtersAvailable = vendor.orderedFilterSubset(VendorFilterGroups.languagesFilters)
-  )
-
-}
-
-@softwareVendorBusinessType(vendor: SoftwareVendorModel) = @{
-
-  customSummaryListRow(
-    key = "business-type",
-    filtersAvailable = vendor.orderedFilterSubset(VendorFilterGroups.businessTypeFilters)
-  )
-
-}
-
-@softwareVendorOperatingSystem(vendor: SoftwareVendorModel) = @{
-
-  customSummaryListRow(
-    key = "operating-system",
-    filtersAvailable = vendor.orderedFilterSubset(VendorFilterGroups.operatingSystemFilters)
-  )
-
-}
-
-@softwareVendorMobileApp(vendor: SoftwareVendorModel) = @{
-
-  customSummaryListRow(
-    key = "mobile-app",
-    filtersAvailable = vendor.orderedFilterSubset(VendorFilterGroups.mobileAppFilters)
-  )
-
-}
-
-@softwareVendorSoftwareType(vendor: SoftwareVendorModel) = @{
-
-  customSummaryListRow(
-    key = "software-type",
-    filtersAvailable = vendor.orderedFilterSubset(VendorFilterGroups.softwareTypeFilters)
-  )
-
-}
-
-@softwareVendorAccessibility(vendor: SoftwareVendorModel) = @{
-
-  customSummaryListRow(
-    key = "accessibility",
-    filtersAvailable = vendor.orderedFilterSubset(VendorFilterGroups.accessibilityFilters)
+    key = "submission-type",
+    filtersAvailable = vendor.orderedFilterSubset(VendorFilterGroups.submissionTypeFilters)
   )
 
 }
 
 @softwareVendorDetail(vendor: SoftwareVendorModel) = @{
 
   Seq(
-    softwareVendorAccessibility(vendor),
     softwareVendorPricing(vendor),
-    softwareVendorSuitableFor(vendor),
-    softwareVendorOperatingSystem(vendor),
-    softwareVendorMobileApp(vendor),
-    softwareVendorSoftwareType(vendor),
     softwareVendorSoftwareFor(vendor),
-    softwareVendorBusinessType(vendor),
-    softwareVendorCompatibility(vendor),
-    softwareVendorLanguage(vendor)
+    softwareSubmissionType(vendor),
+    softwareVendorSuitableFor(vendor)
   ).flatten
 
 }
 
 @softwareVendorListing(vendor: SoftwareVendorModel, index: Int) = {
-  <div id="software-vendor-@index">
+  <div id="software-vendor-@index" style="display: block;">
```

Comment:
```suggestion
  <div id="software-vendor-@index">
```
We shouldn't be adding custom styles ideally, what's this doing?